docs/azmcp-commands.md

```diff
@@ -807,21 +807,22 @@ azmcp azureterraformbestpractices get
 ```
 
 ### Azure Virtual Desktop Operations
+
 ```bash
 # List Azure Virtual Desktop host pools in a subscription
 azmcp virtualdesktop hostpool list --subscription <subscription> \
                                    [--resource-group <resource-group>]
 
 # List session hosts in a host pool
 azmcp virtualdesktop hostpool sessionhost list --subscription <subscription> \
-                                                [--hostpool <hostpool-name> | --hostpool-resource-id <hostpool-resource-id>] \
-                                                [--resource-group <resource-group>]
+                                               [--hostpool <hostpool-name> | --hostpool-resource-id <hostpool-resource-id>] \
+                                               [--resource-group <resource-group>]
 
 # List user sessions on a session host
 azmcp virtualdesktop hostpool sessionhost usersession-list --subscription <subscription> \
-                                                [--hostpool <hostpool-name> | --hostpool-resource-id <hostpool-resource-id>] \
-                                                --sessionhost <sessionhost-name> \
-                                                [--resource-group <resource-group>]
+                                                           [--hostpool <hostpool-name> | --hostpool-resource-id <hostpool-resource-id>] \
```

Comment:
is this spacing intentional ?

Reply:
Yeah, I'm just aligning the parameters for readability.